stream-chat-android-state/src/main/java/io/getstream/chat/android/state/plugin/state/global/GlobalState.kt

```diff
@@ -60,6 +60,8 @@ public interface GlobalState {
      */
     public val banned: StateFlow<Boolean>
 
+    public val queryingChannelsFree: StateFlow<Boolean>
```

Comment:
Why do we need to expose this `StateFlow`? 🤔

Reply:
This is what triggers all the StartStopBuffers in all `UnreadCountLogic`. They all need to have the same trigger